### PR TITLE
Update README.md with new possible functionnalities for the INPTify

### PR DESCRIPTION
**Pull request by moun3imy**

### CODE REVIEW

**Comment by moun3imy** [comment]
thanks Youness ;)

File: README.md [hunk 1]
```diff
@@ -8,7 +8,14 @@ By enities we mean Clubs communication cells, different departments heads, the a
 The type of notifications available is:
   1. Simple posts (with title and body).
   2. Exam marks (every student receives his mark only).
-  3. (feel free to suggest more :grin:)
+  3. (feel free to suggest more :grin:).
```
Comment (Yiabiten) [inline README.md]: remove this line :scissors:

File: README.md [hunk 2]
```diff
@@ -8,7 +8,14 @@ By enities we mean Clubs communication cells, different departments heads, the a
...
+  
+### FUNCTIONNALITIES :
```
Comment (Yiabiten) [inline README.md]: "Functionalities"

File: README.md [hunk 3]
```diff
@@ -8,7 +8,14 @@ By enities we mean Clubs communication cells, different departments heads, the a
...
   1. Simple posts (with title and body).
   2. Exam marks (every student receives his mark only).
-  3. (feel free to suggest more :grin:)
+  3. (feel free to suggest more :grin:).
+  
+### FUNCTIONNALITIES : 
+  1. communicate schedule of courses to teachers and students, and any made update to the schedules.
+  2. critic notifications (exam marks,time schedules,..) must go through a hierarchy validation system. 
+  3. Students((undergraduates,postgraduates, phd students, postdocs)) and school stuff subscribe to favorite research fields.
+  4. anyone recieve only talks' and conferences' information about the subjects are subscribed to.
```
Comment (Yiabiten) [inline README.md]: "receive", detected by @melbarch :grinning:

Comment (melbarch) [inline README.md]: Thanks for the ideas Mounaim :+1: can u plz review it so we can merge :mag:

**Comment by moun3imy** [comment]
fixes added !

**Pull request merged**